cmd/clean.go

```diff
@@ -187,32 +188,25 @@ func checkFilterUsesFreespace(filter *config.FilterConfiguration) bool {
 	}
 
 	// Check all filter conditions
-	for _, expr := range filter.Ignore {
-		if checkExpression(expr) {
-			return true
-		}
+	if slices.ContainsFunc(filter.Ignore, func(s string) bool { return checkExpression(s) }) == true {
```

Comment:
```suggestion
	if slices.ContainsFunc(filter.Ignore, func(s string) bool { return checkExpression(s) }) {
```
`== true` can be omitted on these, since `ContainsFunc` already returns a bool, right? 😃

Reply:
good catch, I just noticed, that I can clean it up further, will push that in a few 😄